labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise02.scala

```diff
@@ -39,29 +47,38 @@ object ListManipulationExercise02 {
    * in a one-liner.
    */
   def separateTheYoungFromTheOld(persons: List[Person]): List[List[String]] = {
-    var youngins: ListBuffer[Person] = new ListBuffer[Person]()
-    var elders: ListBuffer[Person] = new ListBuffer[Person]()
-    var validYoungNames: ListBuffer[String] = new ListBuffer[String]()
-    var validOldNames: ListBuffer[String] = new ListBuffer[String]()
+    def dostuff(people: List[Person]) = people.sortBy(_.age).map(_.firstName)
+    val (youngs, old) = persons.partition(p => p.age < 18)
+    List( dostuff(youngs), dostuff(old) )
```

Comment:
I really wanted to use map but couldn't figure out how to get map to work on the tuple. I guess I could have done `List(youngs, old) map dostuff` but that's not much shorter